site/content/installation/nginx-plus-jwt.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "NGINX Plus Image and JWT Requirement"
```

Comment:
```suggestion
title: "NGINX Plus and JWT requirement"
```

Reply:
The reason I had Image in here is because of the need to create the Docker Registry Secret to pull the NGINX Plus docker image, which is an extra step before creating the JWT token Secret.

Reply:
In context it makes sense, but the page title itself isn't what provides the context, it's the instructions.

We try to prioritize conciseness anywhere we can get it: there might be a better title for the document overall TBH.

It's a super minor LOGAF concern though: I can think about it more in the future.